src/components/thumb-carousel/thumb-carousel.jsx

```diff
@@ -28,17 +29,23 @@ export default React.createClass({
     let triggers = this.props.contents.map((item, index) => {
       let classNames = `hero${this.state.activeTrigger === index ? ` active` : ``}`;
 
+      // Using anchors instead of buttons because Firefox adds "padding" to button elements even with padding:0
       return (
-        <button key={index} className={classNames} onClick={this.onTriggerClick}>
+        <a key={index} className={classNames} onClick={this.onTriggerClick}>
           <img className="photo" src={item.image}></img>
-        </button>
+        </a>
       );
     });
 
     let quotes = this.props.contents.map((item, index) => {
       return (
         <div key={index} className={this.state.activeTrigger === index ? `active` : ``}>
-          <p className="caption">"{item.caption}"</p>
+          <div className="wrapper">
+            <div className="quote quote-left"></div>
+            <p className="caption" dangerouslySetInnerHTML={{__html: DOMPurify.sanitize(item.caption)}}></p>
+            <div className="quote quote-right"></div>
+          </div>
+
           <p className="attribution">– {item.attribution}</p>
```

Comment:
it's not, just thinking of what happens when someone maliciously (or social engineeringly) edits the JSON we consume. Just wondering whether we want to see `&lt;script&gt;alert('lol')&lt;/script&gt;` on the page, or just "nothing" because dompurify cleaned it up for us. That decision shouldn't hold up the PR though